lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/view/FindMealFragment.kt

```diff
@@ -28,48 +41,161 @@ class FindMealFragment : Fragment() {
     private lateinit var swipeRefreshLayout: SwipeRefreshLayout
 
     private val sdk = MealsSDK()
-    private val mealsAdapter = MealsAdapter(listOf())
+    private var distanceUnit: DistanceUnit = DistanceUnit.miles
+    private val mealsAdapter = MealsAdapter(listOf(), distanceUnit)
+
+    private lateinit var fusedLocationClient: FusedLocationProviderClient
+    private val permissionResultParser: PermissionResultParser by lazy {
+        PermissionResultParser()
+    }
+    private var lastLocation: Location? = null
+    private var hasSetupLocationUpdates = false
```

Comment:
When this is in a VM lets see if we can do away with some of these states

Reply:
Sounds good